src/fragment-components/a-tooltip.tsx

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import {
+	Tooltip,
+	TextInput,
+	Dropdown
+} from 'carbon-components-react';
+import { AComponent, ComponentInfo } from './a-component';
+import image from './../assets/component-icons/tooltip.svg';
+import {
+	angularClassNamesFromComponentObj,
+	nameStringToVariableString,
+	reactClassNamesFromComponentObj
+} from '../utils/fragment-tools';
+import { css } from 'emotion';
+
+const preventCheckEvent = css`
+.bx--tooltip__label {
+	pointer-events: none;
+}`;
+
+export const ATooltipSettingsUI = ({ selectedComponent, setComponent }: any) => {
+	const dropdownDirectionItems = [
+		{ id: 'bottom', text: 'Bottom' },
+		{ id: 'top', text: 'Top' },
+		{ id: 'right', text: 'Right' },
+		{ id: 'left', text: 'Left' }
+	];
+
+	return <>
+		<Dropdown
+			label='Direction'
+			titleText='Direction'
+			items={dropdownDirectionItems}
+			selectedItem={dropdownDirectionItems.find(item => item.id === selectedComponent.direction)}
```

Comment:
If user doesn't select anything manually, the selected item shown should be the one that's default for the component.

Unless user specifically selects it, direction shouldn't be set in json.

Reply:
I don't see how this was addressed and seems to still be present.

Reply:
try now

Reply:
Direction is still there 
<img width="433" alt="image" src="https://user-images.githubusercontent.com/9692126/229134411-64cad761-abef-4ad8-8d4e-ae8bc752ff6f.png">
which makes sense since it's set in the deftultComponentObj

Reply:
Updated